### ISSUE TITLE
How to use it?

### ISSUE THREAD

**Issue by Tommixoft** [sebj54/gogs-to-github#2]
Hi, tried to migrate my code from GOGS to GitHub..it created 50 repos..empty repos. From 51st it shows rate limit error.  And now always stops at 51st. No code in repos. Can you include some parameter with array or something to let people add excludes, or includes. Or something..how cna i now continue if application always improts from the begining and always stops at 51st.

You should make it somehow bit GitHUb api friendlier...if this gets secondary limit rated only after 50 requests this means it does something incorrect. Too fast mauybe, you should add sleep or somthing..dont know i do not program in JS. Bu now i have full account of empty repositories....more problem screated than solved.

**Comment by sebj54** [comment]
Hi, I'm sorry if this repository does not what you looked for but unfortunately, this tool only create empty repositories with labels, milestones and issues. You should push manually to each repository after running this tool.

At the time I developed this tool, I did not faced any rate limit, that's why it is not handled. As I do no longer use gogs, I won't fix the issue, sorry for that too.
